llvm/lib/Target/RISCV/RISCVFrameLowering.cpp

```diff
@@ -27,6 +27,11 @@
 
 using namespace llvm;
 
+static cl::opt<bool>
```

Comment:
is it possible to use something other than a `cl::opt`? I know we make heavy use of them to control codegen options, but they tend to get dropped on the floor easily if you have to forward `-mllvm` options to the linker in LTO/ThinLTO builds. This is a constant pain point for a lot of folks who need to support multiple build configurations, like the Linux kernel, Fuchsia, Android, or Chrome.